doc/classes/ExternalTexture.xml

```diff
@@ -1,9 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ExternalTexture" inherits="Texture" version="3.2">
 	<brief_description>
-		Adds support for external textures as defined by https://www.khronos.org/registry/OpenGL/extensions/OES/OES_EGL_image_external.txt
+		Enable OpenGL ES external texture extension.
 	</brief_description>
 	<description>
+		Enable support for the OpenGL ES external texture extension as defined by [url=https://www.khronos.org/registry/OpenGL/extensions/OES/OES_EGL_image_external.txt]OES_EGL_image_external[/url].
+
```

Comment:
The empty line should be removed as each line in the XML is treated as a paragraph, so that'd be an extra `<p></p>`.